cypress/e2e/api.spec.cy.js

```diff
@@ -1,48 +1,127 @@
 /// <reference types="cypress" />
 import { onLandingPage } from "../support/pageObject/landingPage";
 
-let NAME;
-let METHOD;
-let URL;
-let STATUSCODE;
+let name;
+let method;
+let url;
+let statusCode;
+let obj;
 
 describe("API intercept", () => {
   beforeEach("open app", () => {
     cy.log("opening web page");
     onLandingPage.openBasePage();
   });
 
-  it("Tags request", () => {
-    NAME = "Popular Tags";
-    METHOD = "GET";
-    URL = "https://api.realworld.io/api/tags";
-    STATUSCODE = 200;
+   it("Tags mock", () => {
+    name = "tags";
+    method = "GET";
+    url = "https://api.realworld.io/api/tags";
+    obj = {
+      "tags": [
+        "Warsaw",
+        "Krakow",
+        "Lodz",
+        "Wroclaw",
+        "Gdansk",
+        "Szczecin",
+        "Bydgoszcz",
+        "Lublin ",
+        "Bialystok",
+      ],
+    }
+    statusCode = 202;
 
-    onLandingPage.interceptCall(NAME, METHOD, URL, STATUSCODE)
+    onLandingPage.interceptAndMockCallTags(name, method, url, statusCode, obj)
```

Comment:
I still don't see the assertions about what was shown on the page. Just maybe let me leave you a hint. After this mock is applied we need to go to the page, grab rendered elements and check were those elements the same as in mock. Is the CSS properly applied?